server/api/jurisdictions.py

```diff
@@ -727,3 +727,90 @@ def get_discrepancy_counts_by_jurisdiction(election: Election):
             for jurisdiction in election.jurisdictions
         }
     )
+
+
+DiscrepanciesByJurisdiction = Dict[str, Dict[str, Dict[str, Dict[str, int]]]]
+# DiscrepanciesByJurisdiction = {
+#     jurisdictionID: {
+#         batchName: {
+#             contestID: {
+#                 reportedVotes:  {choiceID: int},
+#                 auditedVotes:   {choiceID: int},
+#                 discrepancies:  {choiceID: int},
+#     }
+# }
+
+
+def create_nested_discrepancies_dict():
+    return defaultdict(
+        lambda: defaultdict(lambda: defaultdict(lambda: defaultdict(int)))
+    )
```

Comment:
Based on the way you're using this, I don't think you need the last two levels of `defaultdict`. I think this will suffice:

```py
defaultdict(lambda: defaultdict(dict))
```

Also maybe just inline it since it's only called in one place?

Reply:
Ah yeah, thanks. I initially separated it out anticipating I would also need to use it for the future ballot comparison discrepancy function, but with the shorter init then inline seems fine.